examples/carousel/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
probably worth moving the tsconfig to a common package as well (later)

Reply:
Same here, if we reference the global tsconfig, then we would loose the ability to clone a single example, right?